perf/collector_libpfm.go

```diff
@@ -39,7 +39,7 @@ import (
 type collector struct {
 	cgroupPath         string
 	events             Events
-	cpuFiles           map[string]map[int]readerCloser
+	cpuFiles           map[string]map[uint32]map[int]readerCloser
```

Comment:
Is this change necessary? You created another struct that handles uncore perf events.

Reply:
I thought that it would be a good idea to have a `type` label in all perf metrics. It's useful information, for example, to check if my metric has the same type as provided in the configuration file.